_config.yml

```diff
@@ -51,5 +51,8 @@ exclude_patterns:
   - 'article/article.md'
   - '.venv'
   - 'venv'
+  - 'mcstas-powder.ipynb'
+  - 'reduction-powder.ipynb'
+  - 'analysis-powder.ipynb'
```

Comment:
I think this needs to go further up, on line 20 or something, and should look like
```
  - '3-mcstas/mcstas-powder.ipynb'
  - '4-reduction/reduction-powder.ipynb'
  - '5-analysis/analysis-powder.ipynb'
```